examples/plot_strategy.py

```diff
@@ -0,0 +1,82 @@
+"""
+Tyre strategies during a race
+=============================
+
+Plot all drivers' tyre strategies during a race.
+"""
+
+import fastf1 as f
+import fastf1.plotting as p
+from matplotlib import pyplot as plt
+
+###############################################################################
+# Load the race session and create the plot
+
+session = f.get_session(2022, "Hungary", 'R')
+session.load()
+
+fig, ax = plt.subplots(figsize=(5, 10))
+
+laps = session.laps
+
```

Comment:
Defer figures after every block. Or in this case, just create the figure later as well.